daemon/api_debug.go

```diff
@@ -307,6 +327,8 @@ func getDebug(c *Command, r *http.Request, user *auth.UserState) Response {
 		return getChangeTimings(st, chgID, ensureTag, startupTag, all == "true")
 	case "seeding":
 		return getSeedingInfo(st)
+	case "disks":
```

Comment:
This is implemented via the debug endpoint to avoid importing the gadget package in snap?